docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -504,11 +505,19 @@ public class MockGreetingProfile implements QuarkusTestProfile {
     public String getConfigProfile() { <3>
         return "test";
     }
+
+    @Override
+    public List<TestResourceEntry> testResources() { <4>
+        return Collections.emptyList();
```

Comment:
Maybe something like `return List.of(new TestResourceEntry(MyWireMockServerManager.class));` or similar would make the usage even more clear.

Reply:
Good idea!